frontend/src/components/workspaces/main.css

```diff
@@ -0,0 +1,24 @@
+/* WorkspaceGrid */
+
```

Comment:
It seems to me that this classes are used only in a single componente. Please add them in the related component css file.

Reply:
Changed applied [here](https://github.com/netgroup-polito/CrownLabs/pull/460/commits/77c8d7a732e466080c9e305ba922259e2ecdaa42)